docs/tutorials/simple_prf_simulated.ipynb

```diff
@@ -0,0 +1,2328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ba4ad61e",
+   "metadata": {},
+   "source": [
+    "# How to fit a population receptive field model to simulated data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61af7069",
+   "metadata": {},
+   "source": [
+    "**Author**: Malte Lüken (m.luken@esciencecenter.nl)\n",
+    "\n",
+    "**Difficulty**: Beginner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f5e8de2",
+   "metadata": {},
+   "source": [
+    "In this notebook, we provide a walkthrough on how to fit a population receptive field (pRF) model to simulated data.\n",
+    "\n",
+    "A pRF model maps neural activity in a region of interest in the brain (e.g., V1 in the human visual cortex)\n",
+    "to an experimental stimulus (e.g., a bar moving through the visual field). Here, we use the visual domain as an example,\n",
+    "where the part of the visual field that stimulates activity in the region of interest is the pRF.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d5b9dad",
+   "metadata": {},
+   "source": [
+    "Because prfmodel uses Keras for model fitting, we need to make sure that a backend is installed before we begin.\n",
+    "In this tutorial, we use the TensorFlow backend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "04feb568",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from importlib.util import find_spec\n",
+    "\n",
+    "# Hide tensorflow info messages\n",
+    "os.environ[\"TF_CPP_MIN_LOG_LEVEL\"] = \"1\"\n",
+    "\n",
+    "if find_spec(\"tensorflow\") is None:\n",
+    "    msg = \"Could not find the tensorflow package. Please install tensorflow with 'pip install .[tensorflow]'\"\n",
+    "    raise ImportError(msg)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d25bfff",
+   "metadata": {},
+   "source": [
+    "## Defining the stimulus\n",
+    "\n",
+    "Let's start with the first step: Defining the stimulus. In practice, we recommend that users save the stimulus they use in an experiment to a file and load it to avoid mismatches between experiment and analysis. \n",
+    "Because we use simulated data in this tutorial, we load an example stimulus that is included in the package. \n",
+    "The stimulus simulates a bar moving in different directions through a two-dimensional visual field.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f3e0c6f1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Stimulus(design=array[200, 101, 101], grid=array[101, 101, 2], dimension_labels=['y', 'x'])\n"
+     ]
+    }
+   ],
+   "source": [
+    "from prfmodel.examples import load_2d_bar_stimulus\n",
+    "\n",
+    "num_frames = 200  # Simulate 200 time frames\n",
+    "\n",
+    "stimulus = load_2d_bar_stimulus()\n",
+    "print(stimulus)  # noqa: T201 (print found)"
```

Comment:
to avoid confusion for users unfamiliar with linters, you could consider put the `noqa` for docs notebooks in the pyproject.toml?